packages/kit/src/utils/escape.js

```diff
@@ -1,27 +1,42 @@
-// dict from https://github.com/yahoo/serialize-javascript/blob/183c18a776e4635a379fdc620f81771f219832bb/index.js#L25
-/** @type {Record<string, string>} */
-const escape_json_in_html_dict = {
-	'<': '\\u003C',
-	'>': '\\u003E',
-	'/': '\\u002F',
-	'\u2028': '\\u2028',
```

Comment:
if i understand it correctly \u2028 and \u2029 escapes existed not because they could break the html parser, but because they could break the clientside JSON.parse when reading the script tag content.

Is that still safe?

Reply:
Browsers implemented subsume json around 2018. I forgot we still had to step through JS for the parsing, so it might be worth it to keep them, then, for the sake of older browsers.